plugins/inputs/snmp/snmp_test.go

```diff
@@ -148,100 +167,114 @@ func TestFieldInit(t *testing.T) {
 }
 
 func TestTableInit(t *testing.T) {
+	testDataPath, err := filepath.Abs("./testdata")
+	require.NoError(t, err)
+
 	tbl := Table{
-		Oid: ".1.0.0.0",
+		Oid: ".1.3.6.1.2.1.3.1",
 		Fields: []Field{
 			{Oid: ".999", Name: "foo"},
-			{Oid: "TEST::description", Name: "description", IsTag: true},
+			{Oid: ".1.3.6.1.2.1.3.1.1.1", Name: "atIfIndex", IsTag: true},
+			{Oid: "RFC1213-MIB::atPhysAddress", Name: "atPhysAddress"},
+		},
+	}
+	s := &Snmp{
+		ClientConfig: snmp.ClientConfig{
+			Path: []string{testDataPath},
 		},
```

Comment:
Add the table to the config and run only `s.Init`?

Reply:
The has been cleaned up as your suggestion.